recipes/sdbus-cpp/all/conanfile.py

```diff
@@ -58,18 +58,19 @@ def requirements(self):
         self.requires("libsystemd/253.3")
 
     def validate(self):
+        if self.info.settings.os != "Linux":
+            raise ConanInvalidConfiguration("Only Linux supported")
```

Comment:
You might want to make the name of the package more obvious in the OS check.

```suggestion
            raise ConanInvalidConfiguration(f"{self.name} only supports Linux")
```

Reply:
Good suggestion. Thanks.